maintenance_equipment_sequence/models/maintenance.py

```diff
@@ -63,16 +63,17 @@ def _inverse_seq_number_next(self):
                 sequence = category.sequence_id._get_current_sequence()
                 sequence.sudo().number_next = category.sequence_number_next
 
-    @api.model
-    def create(self, vals):
-        if not vals.get("sequence_id", False):
-            if vals.get("sequence_prefix", False):
-                vals["sequence_id"] = self.sudo()._create_sequence(vals).id
-        else:
-            vals["sequence_prefix"] = (
-                self.env["ir.sequence"].browse(vals["sequence_id"]).prefix
-            )
-        result = super(MaintenanceEquipmentCategory, self).create(vals)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if not vals.get("sequence_id", False):
+                if vals.get("sequence_prefix", False):
+                    vals["sequence_id"] = self.sudo()._create_sequence(vals).id
+            else:
+                vals["sequence_prefix"] = (
+                    self.env["ir.sequence"].browse(vals["sequence_id"]).prefix
+                )
+        result = super().create(vals_list)
         self._compute_equipment_code()
```

Comment:
This code was incorrect previously even before this patch. I think it should be

```suggestion
        result._compute_equipment_code()
```

Reply:
@pedrobaeza Totally agree. This should be called on the newly created recordset. I added this in a separate commit as it addresses another issue. Is this finde for you?